collection/collection/build.gradle

```diff
@@ -64,16 +67,37 @@ kotlin {
                 dependsOn commonMain
             }
 
-            configure([linuxX64Main, macosX64Main, macosArm64Main]) {
-                dependsOn nativeMain
-            }
-
             nativeTest {
                 dependsOn commonTest
             }
 
-            configure([linuxX64Test, macosX64Test, macosArm64Test]) {
-                dependsOn nativeTest
+            darwinMain {
```

Comment:
I'm confused why sourceSet is called darwinMain if it's just for macosx targets :shrug:

Reply:
I guess it's for the future? A suppose there should be ios and others later?

Reply:
because we'll use it for ios as well. technically speaking, the right word is darwin. (see prior thread with @[arkivanov](https://github.com/arkivanov) )